spec/graphql/app_schema_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe do
+  it "has an up to date .graphql file" do
+    current_defn = MagnifierSchema.to_definition
+    printout_defn = File.read(Rails.root.join("app/graphql/schema.graphql"))
+    assert_equal(current_defn, printout_defn, "Update the printed schema with `bundle exec rake dump_schema`")
```

Comment:
Since we're using RSpec, can we amend this from the Test::Unit assertion to an RSpec expectation, e.g.

```ruby
expect(current_defn).to eq printout_defn
```

Reply:
yup, sure thing. missed that, surprised it still showed as fail without the file